easybuild/easyconfigs/t/TALYS/TALYS-1.95-GCCcore-10.3.0.eb

```
@@ -0,0 +1,41 @@
+# Author: Jasper Grimm (UoY)
+easyblock = 'CmdCp'
+
+name = 'TALYS'
+version = '1.95'
+
+homepage = 'https://tendl.web.psi.ch/tendl_2019/talys.html'
+description = "TALYS is a nuclear reaction program."
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+toolchainopts = {'noopt': True}
+
+source_urls = ['https://tendl.web.psi.ch/tendl_2019/talys/']
+sources = [{'download_filename': 'talys.tar', 'filename': SOURCELOWER_TAR_GZ}]
+checksums = ['3a69420b36c2fcd3e433db91ed45d6fcb59558875244cf066f174eaa18840164']
+
+builddependencies = [('binutils', '2.36.1')]
+
+start_dir = 'source'
+
+# Use environment variable to set structure directory path
+local_sed = 'sed -i -e "s|path=.*|call get_environment_variable(\'TALYS_STRUCTURE_DIR\', path)|g" '
+local_sed += ' -e "s|character*132 path|character*256 path|g" '
+local_sed += ' -e "s|//\'abundance|//\'/abundance|g" machine.f && '
```

@jfgrimm escaping the single quote does not work for me. shouldn't that be:
```suggestion
local_sed += ' -e "s|//'abundance|//'/abundance|g" machine.f && '
```
are you sure adding the extra slash is necessary at all?

---

The extra slash is necessary unless you guarantee that the env var always has a trailing /
The escaped single quotes are so that the easyconfig parses correctly. The parsed commandbthat easybuild runs of course doesn't have them

---

(Note that the other two slashes `//` that are not being changed are Fortran string concatenation operators)

---

ah that makes sense, thanks for the clarifying.